README.md

```diff
@@ -21,7 +21,7 @@ The ChartboostExample project includes two targets:
 
 - `ChartboostExample`: Manual dependency management. 
 
- It requires the `Chartboost.framework` to be dropped into the root directory (where this readme is).
+It requires the `Chartboost.framework`, the lastest version is already in the root directory (where this readme is).
```

Comment:
`... readme is) and linked in the Xcode project.` 
To make clear that nothing needs to be done?
Also, "the lastest version" will get outdated very quickly. Maybe something like:
"The `Chartboost.framework` is already in the root directory (where this readme is) and linked to the Xcode project."
?

Reply:
yes, better that way, more clear 👍🙌